mypyc/ir/rtypes.py

```diff
@@ -182,7 +182,7 @@ def __init__(self,
         self.size = size
         # TODO: For low-level integers, they actually don't have undefined values
         #       we need to figure out some way to represent here.
-        if ctype in ('CPyTagged', 'int32_t', 'int64_t'):
+        if ctype in ('CPyTagged', 'int32_t', 'int64_t', 'CPyPtr'):
```

Comment:
`CPY_INT_TAG` is kind of confusing here since the real empty value is a null pointer (`0`). Please change the initializer for `CPyPtr` to 0 (why not also change for `int32_t` and `int64_t` as well). `CPyTagged` should still have `CPY_INT_TAG`.